test/prism_regression/xstring.rb

```diff
@@ -6,3 +6,8 @@
 %x{ruby -e 'puts "Hello, braces!"'}
 %x[ruby -e 'puts "Hello, square brackets!"']
 %x/ruby -e 'puts "Hello, slashes!"'/
+
+`ruby -e 'puts "Hello, #{}"'`
+`ruby -e 'puts "Hello, #{'interpolation'}"'`
+`ruby -e 'puts "Hello, #{foo}"'`
+`ruby -e 'puts "Hello, #{`echo "backticks"`}"'`
```

Comment:
I didn't find it too helpful to have separate files for the interpolated version so I just put them together with normal xstrings. But I'm ok with extracting them too, just let me know!